### DISCUSSION TITLE
Hi mọi người

### DISCUSSION THREAD

**Discussion by duchuypn10a12** [thanhthao10A12/Lop-10A12-Nhom-09#1] [General]
leader='Thao'
teammate1='Thuan'
teammate2='Duy'
print('Chao ' + leader + ',' + teammate1 + ',' + teammate2) @LeQuangThuan10a12 
@TranQuocDuyTVND10A12 
@thanhthao10A12

**Comment by thanhthao10A12** [comment]
hé lu mọi người :}}

**Reply by LeQuangThuan10a12** [reply]
hii nè

**Reply by thanhthao10A12** [reply]
mấy bạn xem cái phần câu hỏi tìm hiểu bài 4 trên MCT chưa z ?

**Reply by duchuypn10a12** [reply]
> mấy bạn xem cái phần câu hỏi tìm hiểu bài 4 trên MCT chưa z ?

_rồi á_
@thanhthao10A12
